src/libaktualizr/package_manager/CMakeLists.txt

```diff
@@ -13,13 +13,35 @@ aktualizr_source_file_checks(${SOURCES} ${HEADERS})
 # Debian backend
 if(BUILD_DEB)
     target_sources(package_manager PRIVATE debianmanager.cc)
+    add_executable(t_packagemanager_deb debianmanager_test.cc)
+    add_dependencies(build_tests t_packagemanager_deb)
+    target_link_libraries(t_packagemanager_deb aktualizr_static_lib ${TEST_LIBS})
+
+    add_test(NAME test_packagemanager_deb COMMAND ${PROJECT_SOURCE_DIR}/tests/run_debian_tests.sh ${CMAKE_CURRENT_BINARY_DIR}/t_packagemanager_deb
+    ${PROJECT_SOURCE_DIR}/tests/test_data/fake_dpkg)
```

Comment:
This too would probably be best split on separate lines, with the following ones indented in some fashion. NBD.